app/graphql/resolvers/business/common.js

```diff
@@ -37,3 +38,49 @@ export const businessAdditionalDetailsUpdateResolver = async (__, { input }, { d
     }
   }
 }
+
+const validateLockUnlockInput = (input) => {
+  if (!input.reason && !input.note) {
+    throw new Error('Reason and/or note are required')
+  }
+}
+
+export const businessLockResolver = async (__, { input }, { dataSources }) => {
+  logger.info('businessLockResolver', { input })
+  validateLockUnlockInput(input)
+
+  const { sbi, ...lockBodyAttributes } = input
+  logger.info('businessLockResolver2', { sbi, lockBodyAttributes })
+  const organisationId = await dataSources.ruralPaymentsBusiness.getOrganisationIdBySBI(sbi)
+  logger.info('lockOrganisation', { organisationId, lockBodyAttributes })
```

Comment:
If these logs are important, should the unlock code have them too? Or were they just for testing?